code/contacts/org.eclipse.scout.contacts.client/src/main/java/org/eclipse/scout/contacts/client/UserForm.java

```diff
@@ -145,7 +145,7 @@ private String createHtmlContent() {
   private IHtmlElement getGravatarImage() {
     String userDomain = CONFIG.getPropertyValue(UserDomainProperty.class);
     if (userDomain == null) {
-      return HTML.imgByIconId(Icons.Person);
```

Comment:
See comment above, use Icons.User instead of PersonSolid (User is an image, PersonSolid a font icon -> they look different)

Reply:
Done